src/core/request.js

```diff
@@ -34,14 +34,15 @@ function Request(remote, command) {
     command: command,
     id: undefined
   };
+  this._submissionTimeout = this.remote.submission_timeout;
```

Comment:
"submission" makes it sound like it might only apply to "submit" for transactions, whereas this also applies for read-only requests. Maybe we should remove "submission"?